spec/mailers/invitation_mailer_spec.rb

```diff
@@ -15,6 +15,10 @@
         expect(subject.to).to eq [params[:respondent_email]]
       end
 
+      it 'includes the requester name and email in the from field' do
+        expect(subject.header[:from].value).to eq "#{params[:requester_name]} <noreply@roqua.nl>"
+      end
```

Comment:
Dit komt te vervallen door je suggestie om naam in subject i.p.v from-veld op te nemen.